vendor/broccoli-fontawesome-pack.js

```diff
@@ -70,7 +70,7 @@ class FontAwesomePack extends Plugin {
        return `
       export {
         ${selectedIcons.join(',')}
-      }  from '@fortawesome/${this.options.pack}/index.mjs'
+      }  from '@fortawesome/${this.options.pack}/index'
```

Comment:
This does work when I try it.

However, zooming out to consider the context, I wonder: is it still necessary to have this conditional logic at all?

Perhaps this would work, a constant return from that function:


```javascript
const packageContents = () => {
  return `
      export {
        ${selectedIcons.join(',')}
      }  from '@fortawesome/${this.options.pack}'
    `
}

```

I thought that maybe this condition was there to handle loading differently for later-version icon packages that do include an `index.mjs`, and perhaps this constant return would fail on older icon packages. However, I tried setting the three free icon package in the `package.json` to `^5` (which resolved to `5.15.4`), confirmed that they do _not_ have the `index.mjs`, and it still built as expected.

So maybe we could simplify this and just have that simpler return? Scratching my head trying to think of what other valid scenarios there may be which would be incompatible with this simpler approach. I'm not thinking of any yet--besides that version 5 case, which works.